<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delaunator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QGroupBox" name="objectsBox">
        <property name="title">
         <string>Объекты</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pointButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rectButton">
           <property name="text">
            <string>Rect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="circleButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Circle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="feild_but">
           <property name="text">
            <string>Feild</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="functionsBox">
        <property name="title">
         <string>Операции</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="femCalc">
           <property name="text">
            <string>Calculation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FemBandesonButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>FemBandeson</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showMesh_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Ф</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showE_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>E</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Ф lines</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="make_1_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add Points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Rect_Mesh_button">
           <property name="text">
            <string>Rect Mesh</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>femCalc</zorder>
        <zorder>FemBandesonButton</zorder>
        <zorder>showE_button</zorder>
        <zorder>pushButton_5</zorder>
        <zorder>pushButton</zorder>
        <zorder>make_1_button</zorder>
        <zorder>Rect_Mesh_button</zorder>
        <zorder>showMesh_button</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="settingsBox">
        <property name="title">
         <string>Триангуляция</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Triangulate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>Add Points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReMesh">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Refinement Mesh</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="dielectric_label">
               <property name="text">
                <string>Dielectric constant 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dielectric_edit">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>K (0&lt;K&lt;1)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ReMeshEdit">
               <property name="text">
                <string>0.1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Dielectric constant 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dielectric_edit_2">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Min angle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mAngle_edit">
               <property name="text">
                <string>30</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="minEdge_label">
               <property name="text">
                <string>Min Edge Length</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="minEdge_edit">
               <property name="text">
                <string>0.01</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="maxEdge_label">
               <property name="text">
                <string>Max Edge Length</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="maxEdge_edit">
               <property name="text">
                <string>50</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
